src/hotspot/share/interpreter/zero/bytecodeInterpreter.inline.hpp

```diff
@@ -42,6 +43,4 @@
 #define VERIFY_OOP(o)
 #endif
 
-# include "bytecodeInterpreter_zero.inline.hpp"
```

Comment:
Does a space between `#` and `include` have any special significance which should be kept throughout reordering?